nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -94,6 +94,13 @@ def get_concat_char_dataset(
 
     manifest_filepaths = config['manifest_filepath']
     datasets = []
+
+    # needed to support validation Concat Datasets that arrive here as
+    # [[dataset1,dataset2]] otherwise ModelPT would interfere
+    if len(manifest_filepaths) == 1 and not isinstance(manifest_filepaths[0], str):
+        logging.warning(f"flattening {manifest_filepaths}")
```

Comment:
Warning -> info and more descriptive info

Reply:
done